gm4_lightning_rods/data/lightning_rods/functions/create_shulker.mcfunction

```diff
@@ -1,3 +1,3 @@
 setblock ~ ~-1 ~ air
-summon shulker ~ ~-1 ~ {ActiveEffects:[{Id:10b,Duration:20,Amplifier:10b,ShowParticles:0b}],Color:16,DeathLootTable:"lightning_rods:shulker"}
+summon shulker ~ ~-1 ~ {ActiveEffects:[{Id:10b,Duration:20,Amplifier:10b,ShowParticles:0b}],Color:15,DeathLootTable:"lightning_rods:shulker"}
```

Comment:
I kinda liked that the shulker was purple (undyed). It made it feel more like the pur-pur was coming alive, rather than being burnt into oblivion.